_includes/tag_list.html

```diff
@@ -1,4 +1,10 @@
 <div id="sidebar">
+  <div id="advertisement">
```

Comment:
@ercohen14 Could you add a condition to show this only for certain tags? What tags should be use @mauro-oto?